### PR TITLE
MEN-5257 Debian 11 tests support.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: scripts/test/run-tests.sh [hunk 1]
```diff
@@ -160,5 +162,29 @@ else
...
+    echo >&2 "Running the uncompressed test"
+    echo >&2 "----------------------------------------"
+    rm -rf deploy
+    gunzip --force "input/Debian-11-x86-64.img.gz"
+    run_convert "release-2" \
+                "input/Debian-11-x86-64.img" \
+                "--config configs/debian-qemux86-64_config $EXTRA_CONFIG" || test_result=$?
+    ret=0
+    test -f deploy/Debian-11-x86-64-qemux86_64-mender.img || ret=$?
+    assert "${ret}" "0" "Expected uncompressed file deploy/Debian-11-x86-64-qemux86_64-mender.img"
```
Comment (lluiscampos) [inline scripts/test/run-tests.sh:184]: This we don't need - it is enough with one uncompressed test for one image

Comment (merlin-northern) [reply scripts/test/run-tests.sh]: removed.

**Comment by lluiscampos** [comment]
As commented privately, I am not confident that using `mkosi` is the right approach, although judging by the JIRA task description and estimate it seems to be the planned thing to do...

We can clear it out tomorrow with @kacf :+1:

**Comment by merlin-northern** [comment]
> As commented privately, I am not confident that using `mkosi` is the right approach, although judging by the JIRA task description and estimate it seems to be the planned thing to do...
> 
> We can clear it out tomorrow with @kacf 👍

right! thank you!

**Review by kacf — APPROVED** [review]
Looks great!

**Pull request merged**